setup.py

```diff
@@ -5,7 +5,7 @@
 setup(
     name=package_name,
     version='1.1.1',
-    packages=[package_name, package_name + '/filters'],
+    packages=find_packages(),
```

Comment:
you need an import: https://github.com/ros2/launch/blob/79515f5ef4d3d1739d9bfb8002c833cfbb98324e/launch/setup.py#L1